packages/cf-component-radio/src/Radio.js

```diff
@@ -1,6 +1,86 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  cursor: theme.cursor,
+  display: theme.display,
+  minHeight: theme.minHeight,
+  paddingLeft: theme.paddingLeft,
+  marginTop: theme.marginTop,
+
+  '&:first-child': {
```

Comment:
This is always going to match `<Input />`. Can you just add a prop `isFirst` to the `<Input />` instead of this first-child selector?